staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2284,6 +2289,25 @@ type SMBiosConfiguration struct {
 	Family       string `json:"family,omitempty"`
 }
 
+type SupportContainerType string
+
+const (
+	// HotplugAttachment is the container resources of the hotplug attachment pod
+	HotplugAttachment SupportContainerType = "hotplug-attachment"
+	// ContainerDisk is the container resources used to attach a container disk to the Virtual Machine
+	ContainerDisk SupportContainerType = "container-disk"
+	// VirtioFS is the container resources used to attach a virtio-fs volume to the Virtual Machine
+	VirtioFS SupportContainerType = "virtio-fs"
+	// SideCar is the container resources for a side car
+	SideCar SupportContainerType = "side-car"
```

Comment:
Gosh naming things is hard, these are my suggestions.  Choose whatever makes the most sense to you.  I don't care very much and won't comment on this anymore unless you call one "banana" or something.  Actually I think that would be funny, go for it:

```
hotplug-disk
container-disk
virtiofs
sidecar
```

Reply:
I don't care that much either, I think I will skip banana though.